frontend/src/components/Analysis/AnalysisReport/index.tsx

```diff
@@ -299,16 +41,18 @@ export function AnalysisReport(props: Props) {
     setPage(0);
   }
 
-  // Create the example table if a bar is selected, empty element if not
-  const exampleTable = createExampleTable(
-    props,
-    activeSystemExamples,
-    setActiveSystemExamples,
-    page,
-    setPage
+  /** Create the example table if a bar is selected, empty element if not */
+  const exampleTable = (
+    <ExampleTable
+      task={props.task}
+      systems={props.systems}
+      activeSystemExamples={activeSystemExamples}
+      setActiveSystemExamples={setActiveSystemExamples}
+      page={page}
+      setPage={setPage}
+    />
```

Comment:
Is there a reason why we have to create `ExampleTable` here and pass it all the way in, as opposed to creating it in `MetricPane`?

Reply:
Never mind... I saw you created another PR for this